src/cbc_sdk/platform/processes.py

```diff
@@ -947,12 +988,23 @@ def _search(self, start=0, rows=0):
                 raise ApiError(f"Failed to get Process Tree: {result['exception']}")
 
     def _perform_query(self):
+        """
+        Iterate over the results of the query.
+
+        Required Permissions:
+            org.search.events(CREATE, READ)
+        """
         for item in self.results:
             yield item
 
     @property
     def results(self):
-        """Save query results to self._results with self._search() method."""
+        """
+        Save query results to ``self._results`` with ``self._search()`` method.
```

Comment:
Seems weird to reference protected functions in the public facing results function

Reply:
That whole summary line is unsatisfactory. I'm completely rewriting it.